Sources/SimplyCoreAudio/Public/Enums/Scope.swift

```diff
@@ -28,12 +28,16 @@ public enum Scope {
     /// side of an object.
     case playthrough
 
-    /// The AudioObjectPropertyElement value for properties that apply to the master
-    /// element or to the entire scope.
+    /// The AudioObjectPropertyElement value for properties that apply to the main
+    /// element or to the entire scope. Using deprecated naming
     case master
 
     /// The wildcard value for AudioObjectPropertySelectors
     case wildcard
+    
+    /// The AudioObjectPropertyElement value for properties that apply to the main
+    /// element or to the entire scope.
+    case main
```

Comment:
Added at bottom of list for potential? binary compatibility.

May be cargo-culting C idiom needlessly (am not _really_ Swift literate); feel free to re-alphabetize if you know better